bin/node/src/flags/p2p.rs

```diff
@@ -318,19 +315,8 @@ impl P2PArgs {
     /// Returns the unsafe block signer from the CLI arguments.
     pub async fn unsafe_block_signer(
         &self,
-        config: &RollupConfig,
         args: &GlobalArgs,
-        l1_rpc: Option<Url>,
     ) -> anyhow::Result<alloy_primitives::Address> {
-        // First attempt to load the unsafe block signer from the runtime loader.
-        if let Some(url) = l1_rpc {
-            let mut loader = RuntimeLoader::new(url, Arc::new(config.clone()));
-            let runtime = loader.load_latest().await.map_err(|e| {
-                anyhow::anyhow!("Failed to load runtime: {} {:?}", e, std::error::Error::source(&e))
-            })?;
-            return Ok(runtime.unsafe_block_signer_address);
-        }
```

Comment:
This we can't do unfortunately. We must load the current value at startup. Could be done with a much-simplified `RuntimeLoader` though.